Sources/AudioPlayer.swift

```diff
@@ -96,8 +104,10 @@ public final class AudioPlayer: ObservableObject {
             self?.setError(error)
```

Comment:
Should we call `didFinishPlaying()` on failure?

Reply:
probably a good idea. I'll update it.